### DISCUSSION TITLE
Library API?

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [Bodmer/TFT_eSPI#1550] [Q&A - General]
Booting up my first project using SPI and TFT displays. The examples here are great. Although I'm wondering...

Where is the "official" API document for drawing shapes, lines, printing, etc. to the display? Or did I totally missing something?

**Comment by Bodmer** [comment]
Haven't got around to that! The basic functions are Adafruit_GFX compatible. Tutorial here:
https://learn.adafruit.com/adafruit-gfx-graphics-library/coordinate-system-and-units

Other than that there are the examples and header files:
https://github.com/Bodmer/TFT_eSPI/blob/master/TFT_eSPI.h#L391-L708

**Reply by Xylopyrographer** [reply]
Thanks. I'll take a look there. And thanks for this library as well 👍😃.
